feature-group-definitions/task-scheduling.yml

```diff
@@ -0,0 +1,13 @@
+spec: https://wicg.github.io/scheduling-apis/
```

Comment:
How about calling this feature just "scheduler"? That's the simplest name for it I can think or. Or "scheduling", if we think that "idle detection" is a precedent.